src/pages/login/index.js

```diff
@@ -6,48 +6,87 @@ import CredentialsCard from "../../components/credentials";
 import "./login.css";
 
 const Login = () => {
-	const { onLogin } = useAuth();
-	const [formData, setFormData] = useState({ email: "", password: "" });
-
-	const onChange = (e) => {
-		const { name, value } = e.target;
-		setFormData({ ...formData, [name]: value });
-	};
-
-	return (
-		<div className="bg-blue login credentialpage">
-			<CredentialsCard
-				title="Login"
-				socialLinksTitle="Or log in with"
-				altButtonTitle="Need an account?"
-				altButtonLink="/register"
-				altButtonText="Sign up"
-			>
-				<div className="login-form">
-					<form>
-						<TextInput
-							value={formData.email}
-							onChange={onChange}
-							name="email"
-							label={"Email *"}
-						/>
-						<TextInput
-							value={formData.password}
-							onChange={onChange}
-							name="password"
-							label={"Password *"}
-							type={"password"}
-						/>
-					</form>
-					<Button
-						text="Log in"
-						onClick={() => onLogin(formData.email, formData.password)}
-						classes="green width-full"
-					/>
-				</div>
-			</CredentialsCard>
-		</div>
-	);
+  const { onLogin } = useAuth();
+  const [formData, setFormData] = useState({ email: "", password: "" });
+  const [errors, setErrors] = useState({ email: "", password: "" });
+
+  const isValidEmail = (emailAddress) => {
+    const emailPattern = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+    return emailPattern.test(emailAddress);
+  };
+
+  const isValidPassword = (password) => {
+    // Example validation: at least 8 characters, 1 uppercase, 1 number, 1 special character
+    const passwordPattern =
+      /^(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/;
+    return passwordPattern.test(password);
+  };
```

Comment:
Correct me if I'm wrong but aren't the password checks to be added by Atakan in his issue? [Issue #11](https://github.com/boolean-uk/team-dev-client-c13/issues/11)

Reply:
Yea